README.md

```diff
@@ -74,19 +74,34 @@ Using Docker has several advantages:
 1. All builds are consistent. When multiple developers each build the same codebase, they should see exactly the same results -- as long as they're using the same or equivalent JDK, Maven, and Docker versions. That's because the containers will be running the same versions of the services on the same operating systems. Plus, all of the tests are designed to connect to the systems running in the containers, so nobody has to fiddle with connection properties or custom configurations specific to their local environments.
 1. No need to clean up the services, even if those services modify and store data locally. Docker *images* are cached, so building them reusing them to start containers is fast and consistent. However, Docker *containers* are never reused: they always start in their pristine initial state, and are discarded when they are shutdown. Integration tests rely upon containers, and so cleanup is handled automatically.
 
+### Configure your docker environment
+
+The Docker Maven Plugin will resolve the docker host by checking the following environment variables:
+
+    export DOCKER_HOST=tcp://10.1.2.2:2376
+    export DOCKER_CERT_PATH=/path/to/cdk/.vagrant/machines/default/virtualbox/.docker
+    export DOCKER_TLS_VERIFY=1
+    
+These can be set automatically if using Docker Machine or something similar.
+
 ### Building the code
 
 First obtain the code by cloning the Git repository:
 
     $ git clone https://github.com/debezium/debezium.git
-    $ cd keycloak
+    $ cd debezium
 
 Then build the code using Maven:
 
     $ mvn clean install
 
-The build starts and uses several Docker containers for different DBMSes. Note that if Docker is not running, you'll likely get an arcane error -- if this is the case, always verify that Docker is running, perhaps by using `docker ps` to list the running containers.
+The build starts and uses several Docker containers for different DBMSes. Note that if Docker is not running or configured, you'll likely get an arcane error -- if this is the case, always verify that Docker is running, perhaps by using `docker ps` to list the running containers.
+
+### Don't have docker running locally for builds?
+
+You can skip the integration tests and docker-builds with the following command:
 
+    $ mvn clean install -DskipIT -P -docker
```

Comment:
Yah! Sounds good to me! Will update it in a bit.
Thanks!

On Thu, Feb 11, 2016 at 4:04 PM, Randall Hauch <notifications@github.com>
wrote:

> In README.md
> <https://github.com/debezium/debezium/pull/12#discussion_r52682708>:
>
> >
> > +    $ mvn clean install -DskipIT -P -docker
>
> How about using a single property to turn off integration tests?
>
> mvn clean install -DskipIT
>
> Could be done by making the docker profile run except when the skipIT is
> set? WDYT?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/debezium/debezium/pull/12/files#r52682708>.
>



-- 
*Christian Posta*
twitter: @christianposta
http://www.christianposta.com/blog
http://fabric8.io